charts/umbrella/Chart.yaml

```diff
@@ -28,38 +28,20 @@ sources:
   - https://github.com/eclipse-tractusx/e2e-testing
 
 type: application
-version: 0.2.2
+version: 0.3.0
 
 dependencies:
-  - alias: certsconsumer
-    name: certs
-    version: 0.1.0
-  - alias: certsprovider
-    name: certs
-    version: 0.1.0
-
-  # edc consumer
-  - alias: edcconsumer
-    name: tractusx-connector-legacy
-    version: 0.5.0-rc1
-    repository: https://eclipse-tractusx.github.io/charts/dev
-  # edc provider
-  - alias: edcprovider
-    name: tractusx-connector-legacy
-    version: 0.5.0-rc1
-    repository: https://eclipse-tractusx.github.io/charts/dev
```

Comment:
It's good to remove certconsumer and certsprovider but maybe it's better comment out all edc consumer and edc provider related things (with a TODO to upgrade the edc to the R23.12), the same is valid for the values file.
It's kind of a pity to loose all the edc enablement from before just because daps isn't relevant anymore.

Reply:
Hey Evelyn, good point, I've added them back with comment and TODO.

Reply:
@almadigabor looks good! but the linter fails

Reply:
Right, fixed the lint issues.